### PR TITLE
Implement page cart

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement page cart

Desktop

localhost_5173_cart (3)

Mobile
localhost_5173_cart (2)

### CODE REVIEW

File: src/pages/Cart/cart.module.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  display: flex;
+  flex-direction: column;
+  background: var(--color-gradient);
+  height: 21.75rem; /* 348px / 16 */
+  padding: 2.5rem 7.5rem; /* 40px / 16 and 120px / 16 */
+  color: var(--color-text-secondary);
+  margin-bottom: 5rem; /* 80px / 16 */
+
+  /* take 100 % width */
+  width: 100vw;
```
Comment (thanhnguyen-agilityio) [inline src/pages/Cart/cart.module.css]: this is a hack.

File: src/pages/Cart/index.tsx [hunk 2]
```diff
@@ -1,5 +1,58 @@
...
+import MainLayout from '../../layouts/MainLayout';
+import { CartItem } from '../../types/cartItem';
+import styles from './cart.module.css';
+
+// example data
+const productDetails: CartItem[] = [
+  {
+    id: 1,
+    image:
+      'https://encrypted-tbn0.gstatic.com/shopping?q=tbn:ANd9GcQhjpAMGZr_n_ABGlri34mUcwi_mpNlTO1_essdAYk9IrlKhT53CbjMqxh44Lke-HDUO4wLmmCVL8Em31KLUql1Wf3ETUvI_28-kkJXb15eM2eOMBFCUUkT7GZ897B2bIDLBqre-D8&usqp=CAc',
```
Comment (thanhnguyen-agilityio) [inline src/pages/Cart/index.tsx]: use shorten link.

File: src/pages/Cart/index.tsx [hunk 3]
```diff
@@ -1,5 +1,58 @@
...
+        <div className={styles.banner}>
+          <Breadcrumb listItems={breadcrumbItems} />
+          <p className={styles.title}>Cart</p>
+        </div>
+        <div className={styles.container}>
+          <p className={styles.titleCart}>Cart Product</p>
+          <div className={styles.group}>
+            <CartList cartItems={productDetails} />
+            <div className={styles.checkout}>
+              <CartSummaryInfo numberProduct={5} onClickCheckoutButton={() => {}} totalPrice={500} />
```
Comment (thanhnguyen-agilityio) [inline src/pages/Cart/index.tsx]: break line for multiple props.

File: src/pages/Cart/index.tsx [hunk 4]
```diff
@@ -1,5 +1,58 @@
...
+
+const breadcrumbItems = [{ label: 'Home', url: '/' }, { label: 'Cart' }];
+
+// example data end
+
 const Cart = () => {
-  return <></>;
+  return (
+    <MainLayout>
+      <div>
```
Comment (thanhnguyen-agilityio) [inline src/pages/Cart/index.tsx]: use `<section>`

File: src/pages/Cart/index.tsx [hunk 5]
```diff
@@ -1,5 +1,58 @@
...
+        <div className={styles.banner}>
+          <Breadcrumb listItems={breadcrumbItems} />
+          <p className={styles.title}>Cart</p>
+        </div>
+        <div className={styles.container}>
+          <p className={styles.titleCart}>Cart Product</p>
```
Comment (thanhnguyen-agilityio) [inline src/pages/Cart/index.tsx]: semantic tag pls

File: src/pages/Cart/cart.module.css (hunk 1 above)
Comment (thanhtrantruong47) [reply src/pages/Cart/cart.module.css]: fixed

File: src/pages/Cart/index.tsx (hunk 5 above)
Comment (thanhtrantruong47) [reply src/pages/Cart/index.tsx]: fixed

File: src/pages/Cart/index.tsx (hunk 4 above)
Comment (thanhtrantruong47) [reply src/pages/Cart/index.tsx]: fixed

File: src/pages/Cart/index.tsx (hunk 3 above)
Comment (thanhtrantruong47) [reply src/pages/Cart/index.tsx]: fixed

File: src/pages/Cart/index.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply src/pages/Cart/index.tsx]: fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**